src/libstd/rand/mod.rs

```diff
@@ -99,10 +99,10 @@
 //!    let between = Range::new(-1f64, 1.);
 //!    let mut rng = rand::thread_rng();
 //!
-//!    let total = 1_000_000u;
```

Comment:
I think it should be possible to just remove most of the suffixes, could you try that?